app/poros/meeting_poro.rb

```diff
@@ -0,0 +1,21 @@
+class MeetingPoro
+  attr_reader :id,
+              :date,
+              :start_time,
+              :end_time,
+              :is_accepted,
+              :purpose,
+              :partner_id,
+              :is_host
+
+  def initialize(meeting, user_id)
+    @id = meeting.id
+    @date = meeting.date
+    @start_time = meeting.start_time.strftime("%I:%M %p")
```

Comment:
ok this is what you were talking about with decoding the additions made by AR. interesting.